module/VuFind/src/VuFind/View/Helper/Bootstrap5/CopyToClipboardButton.php

```diff
@@ -45,18 +45,24 @@ class CopyToClipboardButton extends \Laminas\View\Helper\AbstractHelper
     /**
      * This helper creates button for copying content of an element into clipboard
      *
-     * @param string $elementSelector jQuery selector for element to copy
+     * @param string $elementSelector css selector for element to copy
+     * @param bool   $hideButtonText  controls whether the description of the button's purpose
+     *                                is displayed as text or only with a title attribute
```

Comment:
This can be flush left for consistency with other comments.

```suggestion
     * is displayed as text or only with a title attribute
```